code/ai/aicode.cpp

```diff
@@ -1999,9 +1999,9 @@ void evaluate_object_as_nearest_objnum(eval_nearest_objnum *eno)
 				num_attacking = num_enemies_attacking(OBJ_INDEX(eno->trial_objp));
                 
                 if (!sip->is_big_or_huge() && num_attacking < eno->max_attackers) {
-                    dist *= (float)(num_attacking + 2) / 2.0f;				//	prevents lots of ships from attacking same target
-                }
-				
+						dist *= (float) (num_attacking+2)/2.0f;				//	prevents lots of ships from attacking same target
```

Comment:
Good catch, there seems to be numerous whitespace changes that has no business of being in this PR. I'll see if I can revert such changes.